### ISSUE TITLE
Brainstorm of other UDFs to implement

### ISSUE THREAD

**Issue by imathews** [jrossthomson/bigquery-utils#2]
- Paired t-test
- ANOVA
- ANCOVA
- MANOVA
- (Multiple) Linear regression
- (Multiple) Logistic regression
- Wilcoxon Signed-Ranks test
- Wilcoxon Rank-Sum Test (aka Mann-whitney U test)

**Comment by boaguilar** [comment]
Yes, the following is a list of potential additional statistical tests:
- Benjamini-Hochberg multiple testing correction (implemented as a procedure)
- Fisher exact test ( I implemented a version in javascript, still needs a little bit of testing )
- I have also implemented Wilcoxon Rank-Sum tests 
- Chi Square test
